authors/apps/ratings/views.py

```diff
@@ -0,0 +1,79 @@
+from django.shortcuts import render
+from rest_framework.views import APIView
+from rest_framework import status
+from .models import User
+from rest_framework.permissions import IsAuthenticated
+from .serializers import RatingSerializers
+from rest_framework.exceptions import NotFound
+from authors.apps.articles.models import Article
+from authors.apps.articles.serializers import ArticleSerializer
+from authors.apps.ratings.models import Rating
+from rest_framework.response import Response
+from django.db.models import Avg
```

Comment:
@jennizalwango could you please help me understand what this AVG import does, and where is is being used in this particular file?